.gitbook.yaml

```diff
@@ -27,6 +27,10 @@ redirects:
     alerts.html: alerts.md
     assets: using-kubecost/navigating-the-kubecost-ui/assets.md
     troubleshoot-install: troubleshooting/troubleshoot-install.md
+    env: install-and-configure/install/environment.md
+    install-and-configure/install/custom-prom: install-and-configure/advanced-configuration/custom-prom.md
+    install-and-configure/install/custom-prom: install-and-configure/advanced-configuration/custom-prom/gcp-gmp-integration.md
```

Comment:
Should this one be:
install-and-configure/install/custom-prom**/gcp-gmp-integration**: install-and-configure/advanced-configuration/custom-prom/gcp-gmp-integration.md